scripts/_common.sh

```diff
@@ -365,3 +365,38 @@ ynh_app_changelog () {
         echo "No significative changes from the changelog..." > "${final_changelog}_lite"
     fi
 }
+
+ynh_system_user_add_group() {
+    # Declare an array to define the options of this helper.
+    local legacy_args=uhs
+    local -A args_array=([u]=username= [g]=groups=)
+    local username
+    local groups
+
+    # Manage arguments with getopts
+    ynh_handle_getopts_args "$@"
+    groups="${groups:-}"
+
+	local group
+	for group in $groups; do
+		usermod -a -G "$group" "$username"
+	done
+}
+
+
+ynh_system_user_del_group() {
+    # Declare an array to define the options of this helper.
+    local legacy_args=uhs
+    local -A args_array=([u]=username= [g]=groups=)
+    local username
+    local groups
+
+    # Manage arguments with getopts
+    ynh_handle_getopts_args "$@"
+    groups="${groups:-}"
+
+	local group
+	for group in $groups; do
+		gpasswd -d "$username" "$group"
```

Comment:
o.o

Reply:
I may have shamelessly copied it from https://unix.stackexchange.com/a/29572, while there is a more Debian answer right below. :/

Reply:
Actually... nah. It's fine like that. With `deluser` there's so many ways to f*ck everything up if `$username` or `$group` is unassigned.

(I may or may not have deleted `nobody` user from my system while testing it...)